src/Service/MailTemplate/LinkedTemplateProcessor.php

```diff
@@ -82,11 +82,6 @@ public function addLink($link)
 
     /**
      * Function to add multiple links.
-     *
-     * @param $link
-     *   A PHPWord link object (returned when using addLink).
-     *   These need to be passed to the template processor so the
-     *   links relationship file can be updated.
```

Comment:
Lets keep this. 

I assume some sort of automatic fix couldn't figure out how to fix this so it just removed it? Maybe because the parameter is `$links` not `$link`?

Reply:
Nice catch! Fixed!